### PR TITLE
use renamewithnetsh for nc rename

### PR DESCRIPTION
**Pull request by dejandundjerski**
**Reason for Change**:
Use netsh when creating NC instead of powershell to improve overall create time.

### CODE REVIEW

**Review by tamilmani1989 — APPROVED** [review]

**Review by ramiro-gamarra — CHANGES_REQUESTED** [review]
No change needed per se, but need to clarify what the impact of this change is since it will be default for all CNS users.

File: cns/networkcontainers/networkcontainers_windows.go [hunk 1]
```diff
@@ -160,6 +161,8 @@ func createOrUpdateWithOperation(adapterName string, ipConfig cns.IPConfiguratio
 		"true",
 		"/weakhostreceive",
 		"true",
+		"/renamewithnetsh",
+		fmt.Sprintf("%t", configuration.CnsConfig.EnableRenameWithNetSh),
```
Comment (ramiro-gamarra) [inline cns/networkcontainers/networkcontainers_windows.go:165]: Should not be consumed from a global. It should be passed as a parameter (like `setWeakHost`)

File: cns/configuration/configuration.go [hunk 2]
```diff
@@ -48,8 +48,11 @@ type CNSConfig struct {
 	WatchPods                   bool
 	EnableAsyncPodDelete        bool
 	AsyncPodDeletePath          string
+	EnableRenameWithNetSh       bool
 }
 
+var CnsConfig CNSConfig
```
Comment (ramiro-gamarra) [inline cns/configuration/configuration.go:54]: no globals please.

**Review by ramiro-gamarra — CHANGES_REQUESTED** [review]

**Pull request closed without merging**